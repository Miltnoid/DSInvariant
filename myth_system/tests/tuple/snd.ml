type nat =
  | O
  | S of nat

let snd : nat * nat -> nat |>
 { (0, 1) => 1
 ; (2, 3) => 3
 } = ?
