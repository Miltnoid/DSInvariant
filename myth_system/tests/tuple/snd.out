(* Generated Sun Mar  1 19:57:34 2015 *)
let snd : nat * nat -> nat =
  fun (t1:nat * nat) -> #2 t1
;;
